forte/processors/sentiment_analysis.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__all__ = [
+    "VaderSentimentProcessor",
+]
+
+from texar.tf import HParams
```

Comment:
We can also use `texar.torch` here. I think CI may be failing because of this since we don't install texar-tf in the CI.

Reply:
fixed in 88a8b2b6920e1216d489fda5640cb7be699a4f7c